admin/app/components/solidus_admin/ui/toast/component.js

```diff
@@ -2,16 +2,18 @@ import { Controller } from '@hotwired/stimulus'
 
 export default class extends Controller {
   static targets = ['closeButton']
-  static classes = ['closing']
-  static values = { transition: Number }
+  static values = { transition: Number, animationClasses: String }
```

Comment:
Stimulus provides a specific set of attributes for classes, better to use that instead of values:

https://stimulus.hotwired.dev/reference/css-classes#definitions